.travis.yml

```diff
@@ -6,6 +8,23 @@ python:
 virtualenv:
   system_site_packages: true
 
+services:
+  - docker
+
+env:
+  - BACKEND: 'geonode.geoserver'
+#  FIXME. Will be uncommented when qgis_server backend exists
```

Comment:
The second environment will be uncommented when we roll out geonode.qgis_server to geonode/master.